integrations/hadoop/prometheus_metadata.yaml

```diff
@@ -0,0 +1,44 @@
+platforms:
```

Comment:
This file is missing the following metrics that are present in the [hadoop-prometheus-overview.json](https://github.com/GoogleCloudPlatform/monitoring-dashboard-samples/blob/40b6c1c95cffb0e9180db9180e870f7fcf848663/dashboards/hadoop/hadoop-prometheus-overview.json) dashboard:

- hadoop_namenode_missingblocks

Should that be added here?